guides/duplicate-theming-styles.md

```diff
@@ -0,0 +1,51 @@
+# Avoiding duplicated theming styles
+
+As explained in the [theming guide](./theming.md), a theme in Angular Material consists of
+configurations for the `color`, `density` and `typography` systems. As some of these individual
+systems have default configurations, it could happen that styles are unnecessarily duplicated.
+
+Below is an example of a pattern that causes duplicated theme styles being generated.
+
+```scss
+$light-theme: mat-light-theme((color: ...));
+$dark-theme: mat-dark-theme((color: ...));
+
+// Generates styles for all systems configured in the theme. In this case, color styles
+// and default density styles are generated. Density is in themes by default.
+@include angular-material-theme($light-theme);
+
+.dark-theme {
+  // Generates styles for all systems configured in the theme. In this case, color styles
+  // and the default density styles are generated. **Note** that this is a problem because it
+  // means that density styles are generated *again*, even though only the color should change.
+  @include angular-material-theme($dark-theme);
+}
+```
+
+To fix this, the dedicated mixin for color styles should be included for the `.dark-theme`. Replace
+the `angular-material-theme` include for the dark theme with `angular-material-color`. For example:
```

Comment:
"the `angular-material-theme` mixin and include the dark theme using `angular-material-color`".